collections/Hamilton/MS_Hamilton_49.xml

```diff
@@ -50,16 +43,41 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite: two parts</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="mixed">
+                        <support>paper and parchment (binding fragments)</support>
+                        <extent>362 + 4 (former pastedown) leaves <dimensions unit="mm" type="leaf">
+                              <height>210</height>
+                              <width>150</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding notBefore="1459" notAfter="1500">
+                        <p>Contemporary, quarter-bound in leather over wooden boards with stamped fleur-de-lis designs. Paper labels on upper board, <q>Exposicio epistolarum dominicalium et officiorum misse per circulum anni</q> and <q>C 94</q>. Three clasps, now lacking.</p>
+                     </binding>
+                     <binding notBefore="1850" notAfter="1950">
+                        <p>Rebacked, nineteenth to mid-twentieth century.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notAfter="1475" notBefore="1450">15th century, 3rd quarter</origDate>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName role="fmo" key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName>: in its medieval catalogue with the signature <q>C 94</q> (also labelled on the upper board), as printed by P. Lehmann, <title>MBDS</title> ii. 232–593. </origPlace>
+                  </origin>
+                  <provenance notBefore="1841" notAfter="1856"><persName role="fmo" key="person_54195353">Sir William Hamilton, 1788-1856</persName></provenance>
```

Comment:
add before this: Friedrich Gottlieb Julius von Bülow, 1760-?1831, identifiable as 301 in the catalogue of part 3 of his sale, 10 Oct. 1836 ('Expositio epistolar. dominical: et officiormn missae per circulum anni (a. 1456. scr.) - AUGUSTINI Liber de spiritu et littera etc.')
That catalogue is online https://dfg-viewer.de/show/?tx_dlf[id]=http%3A%2F%2Farchiv.ub.uni-marburg.de%2Feb%2F2016%2F0115%2Fmets-6658.xml and can be downloaded. All or almost all the Erfurt manuscritps were in Bulow's collection before coming to Hamilton.